docker/tomcat.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services:
+    ff-test:
+        volumes:
+            - "../test/src/test/testtool:/opt/frank/testtool-ext"
+        image: private.docker.nexus.frankframework.org/ff-test:${VERSION:-latest}-tomcat
+        build: Tomcat/test
+        ports:
+            - "80:8080"
+        environment:
+            active.authentication: false
```

Comment:
Waarom staat deze uit?

Reply:
Ik heb het overgenomen. Moet die weer aan?

Reply:
@philipsens Die moeten aan ja, en als ze dan niet werken moeten we zorgen die fouten op te lossen. 🙂

Reply:
Is goed!